ts/components/ButtonDefaultOpacity.tsx

```diff
@@ -54,7 +55,7 @@ const ButtonDefaultOpacity = (props: Props) => {
       onPress={tapGestureRequired ? undefined : props.onPress}
       accessible={true} // allows with TalkBack the feedback request to touch for button activation
       accessibilityRole={"button"}
-      hitSlop={{ top: slop, bottom: slop }}
+      hitSlop={{ top: slop, bottom: slop, right: slop, left: slop }}
```

Comment:
We should avoid to make changes with this impact: `ButtonDefaultOpacity` is widely used in the codebase. It could generate a side effect that we can't evaluate.

Maybe a safer approach could be add to it an optional props

`customHitSlop?: Insets`

and use it only when it is defined in place of the computed one

`hitSlop={ customHitSlop ?? { top: slop, bottom: slop}}`

What do you think about this alternative solution?

Reply:
Does it make sense for the `hitSlop` to be applied only on the vertical axis by default? This could become confusing and lead to bugs. Didn't really see any regression when inserting the initial PIN with this change. Since the keypad buttons are laid out side by side, this should be a meaningful regression test.

Let me know how to close this.

Reply:
Since we don't really need to customize `hitSlop`, I've thought of defining a `hasFullHitSlop` prop to avoid repeating `customHitSlop={{ top: slop, right: [...]` in the buttons target of this PR